test/test.externalclient.ts

```diff
@@ -116,6 +129,57 @@ describe('ExternalAccountClient', () => {
       );
     });
 
+    it('should return IdentityPoolClient with expected workforce configs', () => {
+      const validWorkforceIdentityPoolClientAudiences = [
+        '//iam.googleapis.com/projects/123/locations/global/workforcePools/workforcePools/providers/provider',
+        '//iam.googleapis.com/projects/workforcePool/locations/global/workforcePools/pool/providers/provider',
+        '//iam.googleapis.com/projects/projectId/locations/global/workforcePools/workloadPools/providers/oidc',
+      ];
+      const workforceFileSourcedOptions = Object.assign(
```

Comment:
Does this work?

const workforceFileSourcedOptions = Object.assign(
        {},
        fileSourcedOptions as any
        {workforce_pool_user_project: workforce_pool_user_project,